retriever/lib/engine_tools.py

```diff
@@ -19,6 +19,7 @@
 
 from hashlib import md5
 from io import StringIO as NewFile
+from distutils.version import LooseVersion
 from retriever.lib.defaults import HOME_DIR, ENCODING, RETRIEVER_REPOSITORY, RETRIEVER_DATASETS
```

Comment:
remove ENCODING

Reply:
@henrykironde sorry, but ENCODING is used in the file.